taipy/core/notification/notifier.py

```diff
@@ -153,16 +147,20 @@ def unregister(cls, registration_id: str) -> None:
         to_remove_registration: Optional[_Registration] = None
 
         for _, registrations in cls._topics_registrations_list.items():
-            for registration in registrations:
-                if registration.registration_id == registration_id:
-                    to_remove_registration = registration
+            for reg in registrations:
+                if reg.registration_id == registration_id:
```

Comment:
Does this work if the `registration_id` is a `_Registration` object?

Reply:
You are right. This does not work.

IT was an attempt to make it more generic but it is wrong as the `_Registration` should not be exposed. I remove the possibility to have a `_Registration` object as parameter.